xlators/cluster/afr/src/afr-self-heal-entry.c

```diff
@@ -263,6 +285,10 @@ afr_selfheal_recreate_entry(call_frame_t *frame, int dst, int source,
 
     mode = st_mode_from_ia(iatt->ia_prot, iatt->ia_type);
 
+    ret = afr_set_heal_outcast(xdata, outcast, AFR_METADATA_TRANSACTION, 1);
```

Comment:
Could you add a comment that, since this entry is created with this fop, no need to take metadata lock to perform this operation?

Reply:
Done